android/gltfio-android/build.gradle

```diff
@@ -29,6 +29,7 @@ android {
 
     sourceSets {
         main {
+            java.srcDirs += "../filament-android/src/main/java"
```

Comment:
That's not right, you are compiling Filament's base library Java classes into each library. This means you can only use one library at a time. It's fine for now as filament-utils depends on gltfio but it means we can never make a "sibling" library to either of them (for instance an FBX loader).